src/SortMsg.chpl

```diff
@@ -65,7 +68,10 @@ module SortMsg
             return b;
           }
           when SortingAlgorithm.RadixSortLSD {
-            return radixSortLSD_keys(a);
+            var numTasks = if nt != -1 then nt else here.maxTaskPar;
+            var bitsPerDigit = if bpd != -1 then bpd else RSLSD_bitsPerDigit;
+
```

Comment:
for now I would create the plan here calling `const plan = makeRadixSortLSDPlan()` or something.